workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java

```diff
@@ -194,6 +199,33 @@ public WorkFlowStatusResponseDTO getWorkFlowStatus(UUID workFlowExecutionId) {
 				.works(workFlowWorksStatusResponseDTOs).build();
 	}
 
+	@Override
+	public WorkFlowContextResponseDTO getWorkflowParameters(UUID workFlowExecutionId,
+			List<WorkContextDelegate.Resource> params) {
+		WorkFlowExecution workFlowExecution = workFlowRepository.findById(workFlowExecutionId).orElseThrow(() -> {
+			throw new ResponseStatusException(HttpStatus.NOT_FOUND,
+					String.format("workflow execution id: %s not found!", workFlowExecutionId));
+		});
+		Map options = null;
```

Comment:
Is there a way to limit sensitive information leak?

Reply:
Pls create a new issue to track this one, since this issue may exist with or without this change.